corehq/apps/cloudcare/static/cloudcare/js/formplayer/sessions/controller.js

```diff
@@ -1,8 +1,14 @@
 hqDefine("cloudcare/js/formplayer/sessions/controller", function () {
     var FormplayerFrontend = hqImport("cloudcare/js/formplayer/app");
     return {
-        listSessions: function () {
-            var fetchingSessions = FormplayerFrontend.getChannel().request("sessions");
+        listSessions: function listSessions(pageNumber, pageSize) {
+            if (pageSize == null) {
+                pageSize = hqImport("cloudcare/js/formplayer/constants").DEFAULT_INCOMPLETE_FORMS_PAGE_SIZE;
+            }
+            if (pageNumber == null) {
```

Comment:
This is intentional

Reply:
Can you disable the linter for this expression? https://eslint.org/docs/2.13.1/user-guide/configuring#disabling-rules-with-inline-comments

Reply:
Sure!

Reply:
I think this is disabling eslint for the whole file, or at least all code after this line, because there's no ending `eslint-enable` block comment. An inline `// eslint-disable-line eqeqeq` might be better.